### ISSUE TITLE
BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.

### ISSUE THREAD

**Issue by asias** [scylladb/scylla#798]
lein test :only cassandra.batch-test/batch-halves-decommission

I saw a lot of them

```
[Thu Jan 14 10:37:07 2016] WARN  [shard 0] BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.
[Thu Jan 14 10:37:08 2016] WARN  [shard 1] BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.
[Thu Jan 14 10:37:08 2016] WARN  [shard 2] BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.
[Thu Jan 14 10:37:11 2016] WARN  [shard 3] BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.
[Thu Jan 14 10:37:11 2016] WARN  [shard 1] BatchStatement - Batch of prepared statements for jepsen_keyspace.b, jepsen_keyspace.a is of size 0, exceeding specified threshold of 5 by 18446744073709551611.
```

**Comment by tgrabiec** [comment]
@asias Is it before or after commit 6a5f075107cb300ce71d2ca969375301876b3fae ?

**Comment by asias** [comment]
after, I am using current master + my patches.

On Thu, Jan 14, 2016 at 6:41 PM, Tomasz Grabiec <notifications@github.com>
wrote:

> @asias <https://github.com/asias> Is it before or after commit 6a5f075
> <https://github.com/scylladb/scylla/commit/6a5f075107cb300ce71d2ca969375301876b3fae>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scylladb/scylla/issues/798#issuecomment-171607540>.
>



-- 
Asias

**Comment by elcallio** [comment]
I am pretty sure this was fixed by f97eed0c945d5731fbacd0794c53079508ca9f11
